activity.py

```diff
@@ -772,6 +781,9 @@ def _get_buddy(self, cs_handle):
         tp_name, tp_path = pservice.get_preferred_connection()
         conn = TelepathyGLib.Connection.new(
             TelepathyGLib.DBusDaemon.dup(), tp_name, tp_path)
+        conn = {}
+        conn_proxy = dbus.Bus().get_object(tp_name, tp_path)
+        conn = dbus.Interface(conn_proxy, TelepathyGLib.IFACE_CONNECTION)
```

Comment:
This one is not like the other one; here, conn is replaced with the interface, but in line 291 conn is a dictionary to which the interface is added as a keyed value.

Reply:
Thanks, I made it a dictionary before, but then gave a second thought on why use a dictionary if we don't need it?

Reply:
Well, if you want to do it that way; don't initialise the dictionary in line 784.

It's almost the same TextChannelWrapper as used in CollabWrapper.  I don't like diverging these classes still further, if it can be avoided.  Another answer would be "this is almost the way @quozl did it in https://github.com/sugarlabs/collabwrapper/pull/15/files". :grin:

Reply:
Thanks, made the changes. Also forgot to remove the reference to _TelepathyGLib_ operation did that too. :sweat_smile:

Reply:
Reviewed, thanks